app/policies/order_policy.rb

```diff
@@ -18,8 +18,8 @@ def resolve
       elsif access_scopes.include?('user')
         scope.by_owner
       else
-        Rails.logger.error("Error in scope search for #{scope.table_name}")
-        Rails.logger.error("Scope does not include admin, group, or user. List of scopes: #{access_scopes}")
+        Rails.logger.debug("Error in scope search for #{scope.table_name}")
```

Comment:
Should we use the work "Error" in these log lines now that it is a debug message?  Can we come up with a better wording?